src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -1011,9 +1035,28 @@
                 "trap_ids": "ip2me",
                 "trap_group": "queue1_group1"
             }
+        },
+       
+        "MCLAG_DOMAIN": {
```

Comment:
add this configuration in https://github.com/Azure/SONiC/wiki/Configuration as source of truth, if not added already.